### ISSUE TITLE
RequestTrackingAuthorizationWithCompletionHandler hardcoded Adjust scene name

### ISSUE THREAD

**Issue by psstevenchan** [adjust/unity_sdk#203]
Adjust Event allow setting sceneName

 `public void setEventSuccessDelegate(Action<AdjustEventSuccess> eventSuccessDelegate, string sceneName = "Adjust")
 `

But requestTrackingAuthorizationWithCompletionHandler is hardcoded scene name 
 AdjustUnity.mm
 ```
void _AdjustRequestTrackingAuthorizationWithCompletionHandler() {
        [Adjust requestTrackingAuthorizationWithCompletionHandler:^(NSUInteger status) {
            NSString *stringStatus = [NSString stringWithFormat:@"%tu", status];
            const char* charStatus = [stringStatus UTF8String];
            UnitySendMessage([@"Adjust" UTF8String], "GetAuthorizationStatus", charStatus);
        }];
    }
```

**Comment by uerceg** [comment]
Hi @psstevenchan 

That's true and we plan to address this in one of the future releases. With current state of things you should either:
- Use `Adjust.prefab` in your scene (it will make sure that GO is present with `Adjust.cs` script behind it) OR
- Create GO on your own, but make sure to name it `Adjust` and that `Adjust.cs` script is assigned to that GO. For example that might look something like this:
```csharp
var adjustGO = Object.FindObjectOfType<Adjust>();
if (adjustGO != null) {
    Destroy(adjustGO.GetComponent<Adjust>());
}
var adjustEnvironment = AdjustEnvironment.Sandbox; // or AdjustEnvironment.Production;
var adjustConfig = new AdjustConfig(adjustAppToken, adjustEnvironment);
new GameObject("Adjust").AddComponent<Adjust>();
Adjust.start(adjustConfig);
```

In case you have any further questions, feel free to ping us in here.

Cheers

**Comment by uerceg** [comment]
Hi @psstevenchan 

Actually update on this one - we'll address this in our next Unity SDK release and add option to define custom scene name for this callback in same fashion like it's being done for callbacks which are possible to be set on **AdjustConfig** instance.

Will keep you posted on the release.

Cheers

**Comment by gtino** [comment]
Any ETA on the next release? I am stumbling upon the same problem: on iOS 14.4 I call:

`Adjust.requestTrackingAuthorizationWithCompletionHandler(() => { ... });`

and all I get is the following log message in the console:

`default	12:52:34.862034 +0100	myapp	SendMessage: object Adjust not found!`

Thanks!

**Comment by uerceg** [comment]
Hi @gtino 

Aiming to have it live in next couple of days.

Cheers

**Comment by uerceg** [comment]
Hey guys. We just released Unity SDK v4.26.0 update (https://github.com/adjust/unity_sdk/releases/tag/v4.26.0). Change on this topic was that default `string` parameter was added as 2nd parameter of `requestTrackingAuthorizationWithCompletionHandler` which is set to have scene name `"Adjust"`, but you're free to pick your own and hopefully callback should get fired to it now.

Feel free to check it out and let us know if it solved this issue for you.
